### PR TITLE
fixup merge of PR6534

### PR DESCRIPTION
**Pull request by rainers**
remove some temporary changes to win32.mak

### CODE REVIEW

**Comment by rainers** [comment]
@braddr Hi, are there some recent changes to the build setup? It seems it doesn't replace `make` with `gmake` anymore when starting the tests on Windows. Is this sensitive to changes to dmd/win32.mak?

**Comment by rainers** [comment]
IIUC the auto-tester applies its patches before merging, but if the former fails, we cannot get a successful build. Can we force a merge of this PR?

**Comment by rainers** [comment]
@CyberShadow @MartinNowak Can we temporarily disable the auto-tester requirement to merge this?

**Comment by MartinNowak** [comment]
I can still merge it, trying to understand it first though ;).

**Comment by rainers** [comment]
The culprit is this patch: https://github.com/braddr/at-client/blob/master/src/diff-dmd-win64.diff#L44

File: win32.mak [hunk 1]
```diff
@@ -1,10 +1,6 @@
 MAKE=make
 
 auto-tester-build:
-	cd ../druntime
-	git fetch https://github.com/rainers/druntime.git data_ptrref
-	git merge FETCH_HEAD
-	cd ../dmd
```
Comment (MartinNowak) [inline win32.mak]: Why was this hack in the dmd PR in the first place?

**Comment by MartinNowak** [comment]
> The culprit is this patch: https://github.com/braddr/at-client/blob/master/src/diff-dmd-win64.diff#L44

Why isn't the patch applying part of the tester log output?
So many questions ;).

File: win32.mak (hunk 1 above)
Comment (rainers) [inline win32.mak]: To get some test coverage for the pointer references. I wanted to remove it before merging (PR said [WIP] in the title), but Walter hit the merge button unexpectedly. I also expected that ELF/macho-o implementations would need to be added before inclusion.

**Comment by rainers** [comment]
> Why isn't the patch applying part of the tester log output?

Why doesn't the tester use environment variables or command line arguments instead of patches? ;-)

**Comment by rainers** [comment]
Thanks.

**Comment by rainers** [comment]
Argh, tests still fail now. We can now
* merge https://github.com/dlang/druntime/pull/1763
* or disable the testptrref test
* or revert https://github.com/dlang/dmd/pull/6534

**Comment by wilzbach** [comment]
>> Why isn't the patch applying part of the tester's log output?
>> So many questions ;).
> Why doesn't the tester use environment variables or command line arguments instead of patches? ;-)

CC @braddr

**Pull request merged**